nengo_loihi/utils.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
```

Comment:
I'm :-1: on a `utils.py` file. I would say `shift` should go in `discretize.py` and `Profile` either in `hardware/profile.py` or `synapses.py`. Neither location is really ideal, but the presence of a `utils.py` file means we'll get lazy and put anything and everything in here rather than deciding where it should go.